portfolio/src/main/webapp/blogStyle.css

```diff
@@ -0,0 +1,245 @@
+html {
```

Comment:
Anyhow, a lot of this CSS in identical to style.css. My advice is that any styles common to both the homepage and the blog page be put into a single stylesheet (which could be style.css) and be included in the HTML for both pages. Any styles specific to a single page should be put into a page-specific stylesheet.

Any comments below may also apply to style.css.